### ISSUE TITLE
Job is running twice in the morning, but only once in the afternoon

### ISSUE THREAD

**Issue by madishka** [kelektiv/node-cron#243]
Hey guys,
I set up a job that is supposed to run at 8:45 and 13:45 each day. However, when I output some logs it runs twice each morning (8:44:59 and 8:45:00), but runs correctly at 13:45:00. 

Output:
Wed Aug 24 2016 13:45:01 GMT-0400 (EDT) - Beginning to run the job
Wed Aug 24 2016 13:45:01 GMT-0400 (EDT) - The job has now completed.
Thu Aug 25 2016 08:44:59 GMT-0400 (EDT) - Beginning to run the job
Thu Aug 25 2016 08:44:59 GMT-0400 (EDT) - The job has now completed.
Thu Aug 25 2016 08:45:00 GMT-0400 (EDT) - Beginning to run the job
Thu Aug 25 2016 08:45:00 GMT-0400 (EDT) - The job has now completed.

Code:

```
new CronJob('00 45 08,13 * * *', function() {
    // run my job
}, function () {
    CronJob.stop();
}, true);
```

Any ideas?

**Comment by madishka** [comment]
I should note that this has happened consistently for the last 2-3 days.

**Comment by ncb000gt** [comment]
I have some ideas around this in that the timers for node aren't perfectly exact. So, the timers may somehow think it is 8:45 when it isn't and then we roll again after the trigger to determine if the next timeout and it happens to find that it is the next tick... I'd guess that it's a bit coincidental that it isn't ticking twice later in the day. It may also have something to do with whatever else is going on in your node process or on the machine you're running the code on.

This isn't exactly a feature, but I'm diving into some other issues that are similar in that weird things are happening based on the timeouts so fixes there may also appear for you.

Have you seen this happen more recently?

**Comment by jackhftang** [comment]
I am experience the same problem, I set the task to run on 0 8 8 * * *, and in my systemd log (I remove irrelevant information)  
...
Nov 25 08:08:00 example.com node[22262]: 2016-11-25T00:08:00.502Z <doing something>
Nov 25 08:07:59 example.com node[22262]: 2016-11-25T00:07:59.490Z <doing something>
...

**Comment by ncb000gt** [comment]
Closing due to age. We can create a new ticket if still an issue.
